parsers/IN_WE.py

```diff
@@ -53,7 +58,7 @@ def fetch_data(
     assert kind is not None
 
     r = session or Session()
-    payload = {"date": target_datetime.strftime("%Y-%m-%d")}
+    payload = {"date": target_datetime.strftime("%Y-%m-%d"), "Flag": 24}
```

Comment:
```suggestion
    payload = {
        "date": target_datetime.astimezone(tz=TZ).strftime("%Y-%m-%d"),
        "Flag": 24,
    }
```
To make sure all requests are localized.